Source/Internal/API/APIEndpoints.swift

```diff
@@ -58,7 +58,7 @@ internal enum APIEndpoints {
 
         case .virtusize(let region):
             components.host = "static.api.virtusize.\(region.rawValue)"
-            components.path = "/a/aoyama/staging/sdk-webview.html"
+            components.path = "/a/aoyama/latest/sdk-webview.html"
```

Comment:
@akueisara could we have that linked to a setting so it's easy to target `staging` if needed

Reply:
ok, done 39f2cd3